src/landscape/components/LandscapeListMap.js

```diff
@@ -62,7 +68,29 @@ const LandscapesClusters = () => {
           })}
           key={index}
           position={landscape.position}
-        />
+        >
+          <Popup className="landscape-marker-popup" closeButton={false}>
+            <Link
+              variant="h6"
+              component={RouterLink}
+              to={`/landscapes/${landscape.data.slug}`}
+            >
+              {landscape.data.name}
+            </Link>
+            <Typography variant="caption" display="block" sx={{ mb: 1 }}>
+              {landscape.data.location}
+            </Typography>
+            <Link
+              variant="body2"
+              component={RouterLink}
+              to={`/landscapes/${landscape.data.slug}`}
+            >
+              {t('landscape.list_map_popup_link', {
+                name: landscape.data.name,
+              })}
+            </Link>
```

Comment:
suggestion: remove this, since it goes to the same place as the landscape name.

Reply:
@paulschreiber I read explicit intent from Hana to duplicate that information, so I don't think we should remove this, we can improve it later after Hana comes back